src/commands/deleteCommand.ts

```diff
@@ -0,0 +1,76 @@
+import { setTimeout as wait } from "node:timers/promises";
+import { Command } from "@yuudachi/framework";
+import type { InteractionParam, ArgsParam, CommandMethod } from "@yuudachi/framework/types";
+import { PermissionFlagsBits, time, TimestampStyles } from "discord.js";
+import type { DeleteCommandContextCommand } from "../interactions/context/deleteCommand.js";
+import { GITHUB_THREAD_NAME } from "../util/constants.js";
+
+export default class extends Command<typeof DeleteCommandContextCommand> {
+	public constructor() {
+		super(["Delete Command"]);
```

Comment:
(RFC) Suggesting a rename to smt like `Smart delete`

Reply:
i still prefer cleanup or sth along those lines

Reply:
> i still prefer cleanup or sth along those lines

The idea is that u can delete command when u *e.g.* get wrong ddocs

Reply:
Yeah "Smart delete" sounds good in my opinion.

Reply:
I think to comply with Souji too, it could be `Smart cleanup`

Reply:
Okey so what to pick?
- `Smart Delete` 
- `Smart Cleanup`

Reply:
nothing about this is smart, just call it Cleanup  Message or sth along those lines